common/src/View/DuplicateNodesCommand.cpp

```diff
@@ -104,9 +106,16 @@ namespace TrenchBroom {
             void doVisit(const Model::BrushNode*) override  { setResult(false); }
         };
 
-        bool DuplicateNodesCommand::cloneParent(const Model::Node* node) const {
+        /**
+         * Returns whether, for UI reasons, duplicating the given node should also cause its parent to be duplicated.
+         *
+         * At present, this applies when duplicating a brush inside a brush entity.
```

Comment:
I'd remove the "At preset". It makes the comment seem like you are expecting it be outdated ;-)

Reply:
Haha, exactly :) But yeah we can "omit needless words" here and remove it.